src/spikeinterface/core/waveform_extractor.py

```diff
@@ -1928,7 +1941,11 @@ def _load_extension_data(self):
                 if ext_data_file.suffix == ".json":
                     ext_data = json.load(ext_data_file.open("r"))
                 elif ext_data_file.suffix == ".npy":
-                    ext_data = np.load(ext_data_file, mmap_mode="r")
+                    # The lazy loading loading of extension is complicated because if we compute again
+                    # and have a link to the old buffer in window then it fails
+                    #ext_data = np.load(ext_data_file, mmap_mode="r")
+                    # so go back to full loading
+                    ext_data = np.load(ext_data_file)
```

Comment:
```suggestion
                    # The lazy loading of an extension is complicated because if we compute again
                    # and have a link to the old buffer on windows then it fails
                    # ext_data = np.load(ext_data_file, mmap_mode="r")
                    # so we go back to full loading
                    ext_data = np.load(ext_data_file)
```